Voice/AI Agent with NodeJS/index.js

```diff
@@ -0,0 +1,127 @@
+import * as dotenv from "dotenv";
+import fetch from "node-fetch";
+dotenv.config();
+import express from "express";
+import { RestClient } from "@signalwire/compatibility-api";
+const host = process.env.HOST_APP;
+
+const app = express();
+app.use(express.urlencoded({ extended: true }));
+app.use(express.json());
+
+app.post("/", (req, res) => {
```

Comment:
Could we add some comments above every endpoint, to explain what it does?

Like a simplified version of what you have in the guide, which I like a lot